frontend/.prettierignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+build
+dist
```

Comment:
I think there is a chance we might need to prevent Prettier from running on `.html` files, since they use Django-specific syntax. Though it looks like right now Prettier does ignore them. We’ll see I suppose!